index.html

```diff
@@ -22,12 +22,16 @@
   <body>
     <div class="container-1">
     <div class="container">
-      <ul class="navbar">
+       <header class="navbar">
+        <a href="">
         <li class="left">My logo</li>
+      </a>
+      <nav>
         <li class="right">
           <span class="material-symbols-outlined"> menu </span>
         </li>
-      </ul>
+      </nav>
+      </header>
       <div class="main-text">
```

Comment:
- [ ] Kindly use `section` tag instead of `div` tag as suggested by the previous reviewer 😊

Reply:
Where I use section tags in all or which specific.